README.md

```diff
@@ -1,15 +1,102 @@
-Art of Monitoring Example Rails Application
-================
+# Art of Monitoring Example Rails Application
 
-Getting Started
----------------
+In this document we will walk you through the installation steps. This application requires specific version of ruby. And we are going to install it in ubuntu 18.04 and CentOS 8.
 
-Default user is:
+## Installing Rails Application on Ubuntu 18.04 and CentOS 8
+
+### Install Ruby-2.6.3
+Before installing the application we are going to install ruby with RVM (Ruby Version Manager). But first we need to install some dependencies with the following command:
+
+### Step 1 
+
+On Ubuntu 18.04:
+```
+sudo apt-get install gcc autoconf curl g++  gnupg automake bison libc6-dev libffi-dev libgdbm-dev libncurses5-dev libsqlite3-dev pkg-config sqlite3 zlib1g-dev libtool libyaml-dev make libgmp-dev libreadline-dev libssl-dev nodejs -y
+
+```
+
+On CentOS 8:
+```
+sudo dnf install curl gcc bzip2 openssl-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel
+```
+
+### Step 2
+
+Install the key on Ubuntu 18.04 with the following command:
+```
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+```
+On CentOS 8:
+```
+gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+```
+
+### Step 3
+
+Next download the RVM installation script from the project’s website on Ubuntu 18.04 with the following command:
+```
+curl -sSL https://get.rvm.io | bash -s stable
+```
+
+On CentOS 8:
+```
+curl -sSL https://get.rvm.io | bash -s stable
+```
+
+### Step 4
 
-user: user@example.com
-password: changeme
+After the installation is completed you will get the starting source path and it looks like this ```source /home/vagrant/.rvm/scripts/rvm```, or you can run the following command on Ubuntu 18.04 or CentOS 8:
```

Comment:
Will it look like `/home/vagrant/`? I'd mention it will look like whatever user you install it as or words to that effect.